<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>487</width>
    <height>365</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>487</width>
    <height>365</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>487</width>
    <height>365</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgba(0, 0, 0, 0);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>487</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(33, 33, 33);
color: rgb(255, 255, 255)</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="indent">
     <number>7</number>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_clear">
    <property name="geometry">
     <rect>
      <x>244</x>
      <y>61</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
   background-color: rgb(81, 84, 89);
color: white;
border: 0px
}

QPushButton:pressed {
    background-color: rgb(109, 114, 121);
}</string>
    </property>
    <property name="text">
     <string>C</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_plusMinus">
    <property name="geometry">
     <rect>
      <x>305</x>
      <y>61</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
   background-color: rgb(81, 84, 89);
color: white;
border: 0px
}

QPushButton:pressed {
    background-color: rgb(109, 114, 121);
}</string>
    </property>
    <property name="text">
     <string>+/-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_percent">
    <property name="geometry">
     <rect>
      <x>366</x>
      <y>61</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
   background-color: rgb(81, 84, 89);
color: white;
border: 0px
}

QPushButton:pressed {
    background-color: rgb(109, 114, 121);
}</string>
    </property>
    <property name="text">
     <string>%</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_divide">
    <property name="geometry">
     <rect>
      <x>427</x>
      <y>61</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
   background-color: rgb(255, 149, 1);
   color: white;
   border: 0px
}

QPushButton:pressed {
    background-color: rgb(204, 120, 1);
}</string>
    </property>
    <property name="text">
     <string>รท</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>244</x>
      <y>122</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
   background-color: rgb(128, 128, 128);
   color: white;
   border: 0px
}

QPushButton:pressed {
    background-color: rgba(255, 255, 255, 150);
}</string>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_9">
    <property name="geometry">
     <rect>
      <x>366</x>
      <y>122</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
   background-color: rgb(128, 128, 128);
   color: white;
   border: 0px
}

QPushButton:pressed {
    background-color: rgba(255, 255, 255, 150);
}</string>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_multiply">
    <property name="geometry">
     <rect>
      <x>427</x>
      <y>122</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
   background-color: rgb(255, 149, 1);
   color: white;
   border: 0px
}

QPushButton:pressed {
    background-color: rgb(204, 120, 1);
}</string>
    </property>
    <property name="text">
     <string>ร</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_8">
    <property name="geometry">
     <rect>
      <x>305</x>
      <y>122</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
   background-color: rgb(128, 128, 128);
   color: white;
   border: 0px
}

QPushButton:pressed {
    background-color: rgba(255, 255, 255, 150);
}</string>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>244</x>
      <y>183</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
   background-color: rgb(128, 128, 128);
   color: white;
   border: 0px
}

QPushButton:pressed {
    background-color: rgba(255, 255, 255, 150);
}</string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>366</x>
      <y>183</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
   background-color: rgb(128, 128, 128);
   color: white;
   border: 0px
}

QPushButton:pressed {
    background-color: rgba(255, 255, 255, 150);
}</string>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_subtract">
    <property name="geometry">
     <rect>
      <x>427</x>
      <y>183</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
   background-color: rgb(255, 149, 1);
   color: white;
   border: 0px
}

QPushButton:pressed {
    background-color: rgb(204, 120, 1);
}</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>305</x>
      <y>183</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
   background-color: rgb(128, 128, 128);
   color: white;
   border: 0px
}

QPushButton:pressed {
    background-color: rgba(255, 255, 255, 150);
}</string>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_1">
    <property name="geometry">
     <rect>
      <x>244</x>
      <y>244</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
   background-color: rgb(128, 128, 128);
   color: white;
   border: 0px
}

QPushButton:pressed {
    background-color: rgba(255, 255, 255, 150);
}</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>366</x>
      <y>244</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
   background-color: rgb(128, 128, 128);
   color: white;
   border: 0px
}

QPushButton:pressed {
    background-color: rgba(255, 255, 255, 150);
}</string>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_add">
    <property name="geometry">
     <rect>
      <x>427</x>
      <y>244</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
   background-color: rgb(255, 149, 1);
   color: white;
   border: 0px
}

QPushButton:pressed {
    background-color: rgb(204, 120, 1);
}</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>305</x>
      <y>244</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
   background-color: rgb(128, 128, 128);
   color: white;
   border: 0px
}

QPushButton:pressed {
    background-color: rgba(255, 255, 255, 150);
}</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_0">
    <property name="geometry">
     <rect>
      <x>244</x>
      <y>305</y>
      <width>121</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
   background-color: rgb(128, 128, 128);
   color: white;
   border: 0px
}

QPushButton:pressed {
    background-color: rgba(255, 255, 255, 150);
}</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_decimal">
    <property name="geometry">
     <rect>
      <x>366</x>
      <y>305</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
   background-color: rgb(128, 128, 128);
   color: white;
   border: 0px
}

QPushButton:pressed {
    background-color: rgba(255, 255, 255, 150);
}</string>
    </property>
    <property name="text">
     <string>.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_equals">
    <property name="geometry">
     <rect>
      <x>427</x>
      <y>305</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
   background-color: rgb(255, 149, 1);
   color: white;
   border: 0px
}

QPushButton:pressed {
    background-color: rgb(204, 120, 1);
}</string>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_squareRoot">
    <property name="geometry">
     <rect>
      <x>61</x>
      <y>183</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
   background-color: rgb(81, 84, 89);
color: white;
border: 0px
}

QPushButton:pressed {
    background-color: rgb(109, 114, 121);
}</string>
    </property>
    <property name="text">
     <string>ยฒโx</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_cosine">
    <property name="geometry">
     <rect>
      <x>61</x>
      <y>305</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
   background-color: rgb(81, 84, 89);
color: white;
border: 0px
}

QPushButton:pressed {
    background-color: rgb(109, 114, 121);
}</string>
    </property>
    <property name="text">
     <string>cos</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_sine">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>305</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
   background-color: rgb(81, 84, 89);
color: white;
border: 0px
}

QPushButton:pressed {
    background-color: rgb(109, 114, 121);
}</string>
    </property>
    <property name="text">
     <string>sin</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_nthPower">
    <property name="geometry">
     <rect>
      <x>183</x>
      <y>122</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
   background-color: rgb(81, 84, 89);
color: white;
border: 0px
}

QPushButton:pressed {
    background-color: rgb(109, 114, 121);
}</string>
    </property>
    <property name="text">
     <string>xโฟ</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_rightBrackets">
    <property name="geometry">
     <rect>
      <x>183</x>
      <y>61</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
   background-color: rgb(81, 84, 89);
color: white;
border: 0px
}

QPushButton:pressed {
    background-color: rgb(109, 114, 121);
}</string>
    </property>
    <property name="text">
     <string>)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_squarePower">
    <property name="geometry">
     <rect>
      <x>61</x>
      <y>122</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
   background-color: rgb(81, 84, 89);
color: white;
border: 0px
}

QPushButton:pressed {
    background-color: rgb(109, 114, 121);
}</string>
    </property>
    <property name="text">
     <string>xยฒ</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_nthRoot">
    <property name="geometry">
     <rect>
      <x>183</x>
      <y>183</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
   background-color: rgb(81, 84, 89);
color: white;
border: 0px
}

QPushButton:pressed {
    background-color: rgb(109, 114, 121);
}</string>
    </property>
    <property name="text">
     <string>โฟโx</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_leftBrackets">
    <property name="geometry">
     <rect>
      <x>122</x>
      <y>61</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
   background-color: rgb(81, 84, 89);
color: white;
border: 0px
}

QPushButton:pressed {
    background-color: rgb(109, 114, 121);
}</string>
    </property>
    <property name="text">
     <string>(</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_tangent">
    <property name="geometry">
     <rect>
      <x>122</x>
      <y>305</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
   background-color: rgb(81, 84, 89);
color: white;
border: 0px
}

QPushButton:pressed {
    background-color: rgb(109, 114, 121);
}</string>
    </property>
    <property name="text">
     <string>tg</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_cotangent">
    <property name="geometry">
     <rect>
      <x>183</x>
      <y>305</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
   background-color: rgb(81, 84, 89);
color: white;
border: 0px
}

QPushButton:pressed {
    background-color: rgb(109, 114, 121);
}</string>
    </property>
    <property name="text">
     <string>ctg</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_inverse">
    <property name="geometry">
     <rect>
      <x>61</x>
      <y>61</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
   background-color: rgb(81, 84, 89);
color: white;
border: 0px
}

QPushButton:pressed {
    background-color: rgb(109, 114, 121);
}</string>
    </property>
    <property name="text">
     <string>1/x</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_cubePower">
    <property name="geometry">
     <rect>
      <x>122</x>
      <y>122</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
   background-color: rgb(81, 84, 89);
color: white;
border: 0px
}

QPushButton:pressed {
    background-color: rgb(109, 114, 121);
}</string>
    </property>
    <property name="text">
     <string>xยณ</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_cubeRoot">
    <property name="geometry">
     <rect>
      <x>122</x>
      <y>183</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
   background-color: rgb(81, 84, 89);
color: white;
border: 0px
}

QPushButton:pressed {
    background-color: rgb(109, 114, 121);
}</string>
    </property>
    <property name="text">
     <string>ยณโx</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_number_Pi">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>183</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
   background-color: rgb(81, 84, 89);
color: white;
border: 0px
}

QPushButton:pressed {
    background-color: rgb(109, 114, 121);
}</string>
    </property>
    <property name="text">
     <string>ฯ</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_number_e">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>244</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
   background-color: rgb(81, 84, 89);
color: white;
border: 0px
}

QPushButton:pressed {
    background-color: rgb(109, 114, 121);
}</string>
    </property>
    <property name="text">
     <string>e</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_log10">
    <property name="geometry">
     <rect>
      <x>122</x>
      <y>244</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
   background-color: rgb(81, 84, 89);
color: white;
border: 0px
}

QPushButton:pressed {
    background-color: rgb(109, 114, 121);
}</string>
    </property>
    <property name="text">
     <string>logโโ</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Rad_Deg">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>61</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
   background-color: rgb(81, 84, 89);
color: white;
border: 0px
}

QPushButton:pressed {
    background-color: rgb(109, 114, 121);
}</string>
    </property>
    <property name="text">
     <string>Rad</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_factorial">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>122</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
   background-color: rgb(81, 84, 89);
color: white;
border: 0px
}

QPushButton:pressed {
    background-color: rgb(109, 114, 121);
}</string>
    </property>
    <property name="text">
     <string>x!</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_log_y">
    <property name="geometry">
     <rect>
      <x>183</x>
      <y>244</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
   background-color: rgb(81, 84, 89);
color: white;
border: 0px
}

QPushButton:pressed {
    background-color: rgb(109, 114, 121);
}</string>
    </property>
    <property name="text">
     <string>logโ</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_log2">
    <property name="geometry">
     <rect>
      <x>61</x>
      <y>244</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
   background-color: rgb(81, 84, 89);
color: white;
border: 0px
}

QPushButton:pressed {
    background-color: rgb(109, 114, 121);
}</string>
    </property>
    <property name="text">
     <string>logโ</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Rad_Deg">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>43</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
